lib/bundles/inspec-compliance/cli.rb

```diff
@@ -30,26 +30,22 @@ def login(server) # rubocop:disable Metrics/AbcSize
       url = options['server'] + options['apipath']
       if !options['user'].nil? && !options['password'].nil?
         # username / password
-        success, msg = login_legacy(url, options['user'], options['password'], options['insecure'])
+        msg = login_legacy(url, options['user'], options['password'], options['insecure'])
       elsif !options['user'].nil? && !options['token'].nil?
         # access token
-        success, msg = store_access_token(url, options['user'], options['token'], options['insecure'])
+        msg = store_access_token(url, options['user'], options['token'], options['insecure'])
```

Comment:
`store_access_token` returns an array, so now `msg` will be an array (https://github.com/chef/inspec/pull/932/files#diff-e2fd7d8d44ceb659a9c3a3a7984c53a4R237), which should cause the print statement below to print both items:

![screen shot 2016-08-18 at 9 31 02 am](https://cloud.githubusercontent.com/assets/852371/17767180/d31a1296-6526-11e6-929b-80e99e42e5dd.png)